lib/src/main/java/uk/gov/di/ipv/core/library/service/CriResponseService.java

```diff
@@ -37,4 +38,16 @@ public List<CriResponseItem> getCriResponseItems(String userId) {
     public CriResponseItem getCriResponseItem(String userId, String criId) {
         return dataStore.getItem(userId, criId);
     }
+
+    public void persistCriResponse(
+            String userId, String credentialIssuer, String issuerResponse, String oauthState) {
+        CriResponseItem criResponseItem =
+                CriResponseItem.builder()
+                        .userId(userId)
+                        .credentialIssuer(credentialIssuer)
+                        .issuerResponse(issuerResponse)
+                        .oauthState(oauthState)
+                        .build();
+        dataStore.create(criResponseItem, ConfigurationVariable.CRI_RESPONSE_TTL);
```

Comment:
Should this store the dateCreated and/or expirationTime?

Reply:
Yup, my bad: I missed the dateCreated (now added). We have no immediate plans for the expirationTime so I've removed it from the class - it was there from the ctrl-c/v of the user credentials store. The 14-day table ttl will handle the aging out of the pending credentials.